### DISCUSSION TITLE
GPS positioning problem in Lambert 93

### DISCUSSION THREAD

**Discussion by geguill** [opengisch/QField#4258] [Q&A]
Hrllo,
I am testing GPS location on QField windows with a simulator: Virtual GPS.
If my Qfield project is in EPSG 4326, no problem.
If it is in EPSG 2154 (Lambert 93), the GPS position and the shp layers (also in 2154) are very off.
Do I always have to work in EPSG 4326?
THANKS
Cordially
Gerard

**Comment by nirvn** [comment]
@geguill , can you define very off?

**Comment by geguill** [comment]
Sorry.
GPS position is in the wrong place relative to the layers.

**Reply by nirvn** [reply]
Heh, that was clear already ;) are we talking about 1 meter off or "the other side of the globe" off?

**Reply by geguill** [reply]
About 5000 km to the south

**Reply by nirvn** [reply]
@geguill , ok, if you can attach a test project with a video that'd be of help 

I'd try to check whether an epsg 4236 point layer reprojects properly in your epsg 2154 project within QGIS first. If there's a problem there, it's likely a project setting issue.

**Reply by geguill** [reply]
[essai_5.zip](https://github.com/opengisch/QField/files/11398871/essai_5.zip)

**Reply by Exebecce** [reply]
Hi,
I have the exact same probleme.
[QFIELD Lons.zip](https://github.com/opengisch/QField/files/12095099/QFIELD.Lons.zip)

**Reply by ludodo** [reply]
Hi,
Do you have news about this problem ? It makes a while I didn't use Qfield and I've got the same problem with the 2.8.7 version and Lambert 93 project. My vectorial data are in lambert 93, the basemap is also generated in Lambert 93, but when activating the GPS, this one is in WGS84 !
So I can't see the data while activating the GPS off course.
In oldest version, I didn't have such a problem.
[test_stavertin_230905.zip](https://github.com/opengisch/QField/files/12523758/test_stavertin_230905.zip)

**Reply by ludodo** [reply]
To complete, I downgrade Qfield to version 2.7.6 and it works great, so something has been broken.

**Reply by nirvn** [reply]
@ludodo , could you try to provide us with NMEA logs of your device from QField? 

Documentation on how to do that is found here: https://docs.qfield.org/how-to/gnss/#external-receiver-log

**Reply by ludodo** [reply]
I do not use an external receiver. so I don't have the option proposed in the positionning window.
On one off the device, installing a version 2.7.6 works, and then, update to 2.8.7 doesn't brake the resolution.
But on anther device (crosscall), it doesn(t work with any version.

**Comment by Exebecce** [comment]
For me, the problem has been solved by deleting Qfield from the smartphone and installing it back. 

Le 5 septembre 2023 16:31:59 GMT+02:00, Mathieu Pellerin ***@***.***> a écrit :
>@ludodo , could you try to provide us with NMEA logs of your device from QField? 
>
>Documentation on how to do that is found here: https://docs.qfield.org/how-to/gnss/#external-receiver-log
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/opengisch/QField/discussions/4258#discussioncomment-6915275
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
Fabien Leprince

**Reply by ludodo** [reply]
Not for us, and on twos different devices
